keps/sig-release/2572-release-cadence/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: Defining the Kubernetes Release Cadence
+kep-number: 2572
+authors:
+  - "@justaugustus"
+owning-sig: sig-release
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+status: implementable
+creation-date: 2021-01-21
+reviewers:
+  - "@BenTheElder"
+  - "@derekwaynecarr"
+  - "@dims"
+  - "@hasheddan"
+  - "@jeremyrickard"
+  - "@johnbelamaric"
+  - "@spiffxp"
+  - "@stevekuznetsov"
+approvers:
+  - "@LappleApple"
+  - "@saschagrunert"
```

Comment:
IMO we need approvers from more than just sig release on this

Arch, api machinery, and node come to mind as primarily impacted by what this means for api changes, deprecation policies and skew support

Testing as another horizontal sig that is impacted by this change

Reply:
@spiffxp -- We agreed in a SIG Leads meeting that SIG Release had `/approve` here, with review from SIG Arch + Testing.

Happy to move a rep from each into the `approver` column, but not to expand the participating groups beyond was already agreed to.

Everyone is welcome to leave feedback and `milestone-maintainers` are tagged here, but the target reviewers/approvers are listed here: https://github.com/kubernetes/enhancements/pull/2567#issuecomment-808493443

Reply:
Do you have a link to meeting minutes recording this decision?

Reply:
@spiffxp -- I believe this was decided in the 1/12 leads meetings, which is linked in the Implementation History section and bookmarked here: https://docs.google.com/document/d/1Jio9rEtYxlBbntF8mRGmj6Q1JAdzZ9fTDo3ru1HK_LI/edit#bookmark=id.val5alfdahlr

Reply:
I'm having trouble finding a written record of the decision in there?

Reply:
fwiw, I am fine with this KEP from a SIG Arch and SIG Node perspective if it addresses the following:

1) we will ship 3 releases per year instead of 4
2) here is what a proposed cadence looks like with 3 releases

other issues related to operation of sig-release itself would defer to sig-release.

would let @dims @johnbelamaric @dchen1107 weigh in as well.

Reply:
I agree with @derekwaynecarr 's take on this.

If there is language in any of our docs around skews that will be affected, please do a quick scan and point them out as well please.

Reply:
@justaugustus The chairs/TL monthly meeting is designed for information sharing, and discussion. It is not intended as a decision making vehicle as there are two meetings (not everyone can attend both), and they are unrecorded.

Reply:
Thank you for highlighting that @cblecker, we've taken that feedback and will ensure future things are not approached in a similar manner.